utils/polus-python-template/{{cookiecutter.container_name}}/plugin.json

```diff
@@ -0,0 +1,59 @@
+{
+    "name": "{{cookiecutter.plugin_name}}",
+    "version": "{{cookiecutter.plugin_version}}",
+    "title": "{{cookiecutter.plugin_name}}",
+    "description": "{{cookiecutter.plugin_description}}",
+    "author": "{{cookiecutter.author}} ({{cookiecutter.author_email}})",
+    "institution": "National Center for Advancing Translational Sciences, National Institutes of Health",
+    "repository": "https://github.com/labshare/polus-plugins",
+    "website": "https://ncats.nih.gov/preclinical/core/informatics",
+    "citation": "",
+    "containerId": "{{cookiecutter.container_id}}:{{cookiecutter.container_version}}",
+    "baseCommand": ["python3", "-m", "{{cookiecutter.plugin_package}}"],
+    "inputs": {
+      "inpDir": {
+          "type": "collection",
+          "title": "Input collection",
+          "description": "Input image collection to be processed by this plugin.",
+          "required": "True"
+      },
+      "filePattern": {
+          "type": "string",
+          "title": "Filename pattern",
+          "description": "Filename pattern used to separate data.",
+          "required": "False"
+      },
```

Comment:
Since this is not a required input, it should have a default value.

Reply:
Can you defined a default value in a wipp manifest? The typer cli implementation do define one ".*"